src/systems/filecoin_mining/storage_proving/sealer/sealer.go

```diff
@@ -77,18 +78,18 @@ func (s *SectorSealer_I) CreateSealProof(si CreateSealProofInputs) *SectorSealer
 	sdr := filproofs.WinSDRParams(cfg)
 	proof := sdr.CreateSealProof(randomSeed, auxTmp)
 
-	onChain := sector.OnChainSealVerifyInfo_I{
-		SealedCID_: auxTmp.CommR(),
+	onChain := abi.OnChainSealVerifyInfo{
```

Comment:
also looks like a lot of fields missing here

Reply:
Ditto